apps/trackers/api.py

```diff
@@ -48,6 +48,7 @@ def post(self, request, *args, **kwargs):
             affects = selected_affects.filter(
                 affectedness=Affect.AffectAffectedness.AFFECTED,
                 resolution__in=[
+                    # TODO: Most probably, only DELEGATED should be here, see OSIDB-2143
```

Comment:
Good point. I thinking about it and even though there should be no new affects with FIX resolution (which is on validations) and it is probably not very common to file trackers against old affects, there is no issue in letting the users to do so.

Reply:
OK, I'll remove the TODO line then.